sdks/python/apache_beam/dataframe/doctests_test.py

```diff
@@ -63,6 +63,20 @@
 ignored result
 '''
 
+ERROR_RAISING_NAME_ERROR_TESTS = '''
+>>> import apache_beam
+>>> raise apache_beam.dataframe.frame_base.WontImplementError('anything')
+ignored exception
+>>> raise NameError
+ignored exception
+>>> undefined_name
+ignored exception
+>>> 2 + 2
```

Comment:
Why should running one line successfully reset `_seen_wont_implement`?

Reply:
I don't want to ignore the whole rest of the docstring, which often has other examples.

Reply:
That sounds fine. If we encounter an issue like this example simulates, I assume we can always manually add the test to the skip map.